src/tbp/monty/frameworks/models/motor_system.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2025 Thousand Brains Project
+#
+# Copyright may exist in Contributors' modifications
+# and/or contributions to the work.
+#
+# Use of this source code is governed by the MIT
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/MIT.
+
+
+from typing import Literal
+
+from tbp.monty.frameworks.actions.actions import Action
+from tbp.monty.frameworks.models.motor_policies import MotorPolicy
+
+
+class MotorSystem:
```

Comment:
Should this class have an abstract method called `set_policy(policy)` for switching the policy? Or is this something you will add later?

Reply:
We'll add this later once we have an idea of the interface's shape. In the meantime, if we have to get work done, we can set `_policy` directly and feel bad about it 🙂.